### PR TITLE
Updates to readme adding documentation.

### PR DESCRIPTION
**Pull request by bweston92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,81 @@ ZeroMQ message dispatcher for ProophServiceBus
...
+        ]
+    ]
+];
+
+$factory = Prooph\ServiceBus\Message\ZeroMQ\Container\ZeroMQMessageProducerFactory;
+$zmqProducer = $factory($container);
+
+// Setup complete, now to add it to the prooph service bus.
+
+// @codeliner can yo give this implementation.
```
Comment (prolic) [inline README.md]: lol, i little more perhaps? or should @codeliner take this issue?

Comment (bweston92) [inline README.md]: @prolic I don't actually know how to utilize the query bus yet.

Comment (codeliner) [inline README.md]: @bweston92 query bus set up is the same as the command bus set up just with a [QueryRouter](https://github.com/prooph/service-bus/blob/master/docs/plugins.md#proophservicebuspluginrouterqueryrouter)

**Comment by codeliner** [comment]
great job :+1: just finish the query bus example and we are done with the docs. More is not needed atm.

**Comment by codeliner** [comment]
@bweston92 any news on the missing part (see line notes)?

**Comment by bweston92** [comment]
@codeliner will do this tonight if possible.

**Comment by codeliner** [comment]
perfect, will perform some final tests and then we can release 1.0 I guess

**Comment by codeliner** [comment]
really good docs, thx

**Pull request merged**